CRAMcodecs.tex

```diff
@@ -453,10 +452,10 @@ \subsubsection*{Order-1 encoding}
 \hline
 \textbackslash0 & a & 4095 \\
 \hline
-a & a &  646 \\
-  & b & 1725 \\
-  & c &  862 \\
-  & d &  862 \\
+a & a &  614 \\
+  & b & 1638 \\
+  & c &  819 \\
+  & d & 1024 \\
```

Comment:
```suggestion
a & a &  614 \\
  & b & 1639 \\
  & c &  819 \\
  & d & 1023 \\
```

I know the spec doesn't define the adjustment to the normalized sum,[^1] but consider being consistent with the (implicit) method used in the order-0 example, i.e., the symbol with the highest frequency is adjusted.

[^1]: § 2.1 "[Frequency table](https://samtools.github.io/hts-specs/CRAMcodecs.pdf#subsection.2.1)" (2023-03-15): "Given rounding differences when renormalising to a fixed sum, it is up to the encoder to decide how to distribute any remainder or remove excess frequencies."